doc/api/Creating_a_Player.md

```diff
@@ -291,6 +291,20 @@ This option will have no effect for contents loaded in <i>Directfile</i>
 mode (see <a href="./Loading_a_Content.md#transport">loadVideo options</a>).
 </div>
 
+### maxBufferSize
+
+_type_: `Number|undefined`
+
+_defaults_: `Infinity`
+
+Set the maximum size of the buffer in the memory, in kilobytes (kb).
+Once this value is reached, the player won't try to download new
+segments anymore
+
+<div class="warning">
+This option will have no effects if we didn't buffer at least <b>MIN_BUFFER_LENGTH</b> 
+<i>( defaults at 5sec )</i>
+</div>
```

Comment:
Same here, we may want to add the directfile warning, that it's only about video buffer and segments, and that the limit is approximative.

Reply:
Still true